feature-libs/quote/core/facade/quote.service.ts

```diff
@@ -554,4 +555,21 @@ export class QuoteService implements QuoteFacade {
       map((quote) => quote as Quote)
     );
   }
+
+  downloadAttachment(
+    quoteCode: string,
+    attachmentId: string
+  ): Observable<Blob> {
+    return this.authService.isUserLoggedIn().pipe(
```

Comment:
MAJOR: should we check if user is logged in before download? I assume user can see the quote details with download button only if they are logged in. So i guess no need to check it again here

Reply:
Ok, updated.